src/EPR.Calculator.Frontend/Constants/StaticHelpers.cs

```diff
@@ -10,5 +10,7 @@ public class StaticHelpers
         public const string MimeType = "test/csv";
         public const long MaxFileSize = 50 * 1024; // 50 KB
         public const string MediaType = "application/json";
+        public const long MaxUploadFileSize = 2 * 1024 * 1024; // 2MB
+        public const string UploadFileNotExceed2MB = "The CSV must be smaller than 2MB";
```

Comment:
Can we remove this `UploadFileNotExceed2MB` as its not being used?

Reply:
Code updated, thanks.